keps/sig-contributor-experience/0005-community-site.md

```diff
@@ -0,0 +1,104 @@
+---
+kep-number: 5
+title: Contributor Site
+authors:
+  - "@jbeda"
+owning-sig: sig-contributor-experience
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: "2018-02-19"
+last-updated: "2018-02-19"
+status: provisional
+---
+
+# Contributor Site
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Stories [optional]](#user-stories-optional)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+    * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Drawbacks [optional]](#drawbacks-optional)
+* [Alternatives [optional]](#alternatives-optional)
+
+## Summary
+
+We need a way to organize and publish information targeted at contributors.
+In order to continue to scale the Kubernetes community we need a convenient, scalable and findable way to publish information.
+
+## Motivation
+
+While the current kubernetes.io site is great for end users, it isn't often used by or aimed at project contributors.
+Instead, most contributors look at documentation in markdown files that are spread throughout a wide set of repos and orgs.
+It is difficult for users to find this documentation.
+
+Furthermore, this documentation is often duplicated and out of date.
+The fact that it isn't collected in one place and presented as a whole leads to fragmentation.
+Often times documentation will be duplicated because the authors themselves can't find the relevant docs.
+
+Finally, some simple domain specific indexing could go a long way to make it easier to discover and cross link information.
+Specifically, building a site that can take advantage of the KEP metadata will both make KEPs more discoverable and encourage those in the community to publish information in a way that *can* be discovered.
+
+### Goals
+
+* Provide a new site (`community.kubernetes.io`).
+* Publish information that is currently in the [community repo](https://github.com/kubernetes/community).
+* Build some simple tools to enhance discoverability within the site.
+  This could include features such as automatically linking KEP and SIG names.
+
+### Non-Goals
+
+* Discover and bring together information from multiple orgs/repos.
+* Create a super dynamic back end.  This is most likely served best with a static site.
```

Comment:
Some basic notes to recommend implementation maybe helpful. I hear that SIG-docs are looking to move `k8s.io` to Hugo, perhaps `community.k8s.io` could pioneer use of Hugo instead of copying anything from docs repo.

Reply:
I'm avoiding going too deep into implementation.  The first step of a KEP is to have the goals outlined and merge that fast.  We can cover the *how* in a follow on PR.

Reply:
Implementation will be determined by goals and tool availability; that belongs in the detail of the KEP, but shouldn't be in the goals section, as Joe says.